core/src/test/resources/application.conf

```diff
@@ -11,3 +11,5 @@ database = {
 messaging {
   mode = "test"
 }
+
+packages.versionFormat = ".+"
```

Comment:
Yeah, it's awkward. What I really want is to only enable it for the Campaign tests, but this isn't possible. Luckily we do not rely on semantic versioning for any other tests in Core...